src/pfe/portal/pushLocal.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+docker cp config/. codewind-pfe:/portal/config
+docker cp middleware/. codewind-pfe:/portal/middleware
+docker cp modules/. codewind-pfe:/portal/modules
+docker cp routes/. codewind-pfe:/portal/routes
+docker cp server.js codewind-pfe:/portal/server.js
```

Comment:
Do you need to add the `openapi.yml` file here? It's copied in when we do a build and is actually used by the `validateReq` middleware function?